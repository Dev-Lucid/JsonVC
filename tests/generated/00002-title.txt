{"title":"title testing","description":"","keywords":"","test":"","author":"","js":"","prepend":[],"append":[],"replace":[]}