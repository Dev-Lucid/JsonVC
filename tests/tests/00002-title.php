<?
jvc::set_response('title','title testing');
file_put_contents($output_path,json_encode($__jvc['response']));
?>