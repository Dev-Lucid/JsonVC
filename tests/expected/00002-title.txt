{"title":"title testing","description":"","keywords":"","author":"","js":"","prepend":[],"append":[],"replace":[]}